docs/graphql/manual/deployment/postgres-requirements.rst

```diff
@@ -86,9 +87,32 @@ Here's a sample SQL block that you can run on your database to create the right
     GRANT ALL ON ALL FUNCTIONS IN SCHEMA public TO hasurauser;
 
     -- Similarly add this for other schemas, if you have any.
-    -- GRANT USAGE ON SCHEMA <schema-name> TO hasurauser;
-    -- GRANT ALL ON ALL TABLES IN SCHEMA <schema-name> TO hasurauser;
-    -- GRANT ALL ON ALL SEQUENCES IN SCHEMA <schema-name> TO hasurauser;
+    GRANT USAGE ON SCHEMA <schema-name> TO hasurauser;
+    GRANT ALL ON ALL TABLES IN SCHEMA <schema-name> TO hasurauser;
+    GRANT ALL ON ALL SEQUENCES IN SCHEMA <schema-name> TO hasurauser;
+    GRANT ALL ON ALL FUNCTIONS IN SCHEMA <schema-name> TO hasurauser;
+
+Note for managed databases (AWS RDS, GCP Cloud SQL, etc.)
+---------------------------------------------------------
+
+Hasura works out of the box with the default superuser, usually called "postgres", created by most managed cloud database providers. But if you are creating a new user and giving the :ref:`above <postgres_permissions>` privileges, then you may notice that the following commands may throw warnings/errors:
+
+.. code-block:: sql
+
+  postgres=> GRANT SELECT ON ALL TABLES IN SCHEMA information_schema TO hasurauser;
+  WARNING:  no privileges were granted for "sql_packages"
+  WARNING:  no privileges were granted for "sql_features"
+  WARNING:  no privileges were granted for "sql_implementation_info"
+  ERROR:  permission denied for table sql_parts
+
+  postgres=> GRANT SELECT ON ALL TABLES IN SCHEMA pg_catalog TO hasurauser;
+  ERROR:  permission denied for table pg_statistic
+
+You can **ignore** these warnings/errors or skip granting these permission as all users have relevant access to ``information_schema`` and ``pg_catalog`` schemas by default (see keyword `PUBLIC <https://www.postgresql.org/docs/current/sql-grant.html>`_) .
+
+.. admonition:: Google Cloud SQL
+
+   On Google Cloud SQL, running ``ALTER SCHEMA hdb_catalog OWNER TO hasurauser;`` may give you an error ``ERROR:  must be member of role "hasurauser"``. You can fix this by running ``GRANT hasurauser to postgres;`` first, assuming "postgres" is the superuser that you are running the commands with. 
```

Comment:
Can this be made kept consistent with the above section. I dont see a need for this to be a note without the code-blocks just because this is specific for Google Cloud SQL. We can just say, "In **Google Cloud SQL** you might see the following error:" followed but the code-block style we have used above